FactoringImp.java

```diff
@@ -0,0 +1,27 @@
+import java.util.LinkedList;
+import java.util.List;
+public class FactoringImp {
+    private String delimiter = ";";
+    public List<String> findFactors(int [] nums){
+        List<String> factors = new LinkedList<>();
+        for(int i : nums) {
+            String s = "";
+            factors.add(findFactorsHelper(i,s));

```

Comment:
Line 9 Should be changed to factors.add(findFactorsHelper(i));
Line 10 Should include factors.add(delimiter)